### ISSUE TITLE
"VirtualizedLists should never be nested inside plain ScrollViews with the same orientation - use another VirtualizedList-backed container instead."  while tap item picker

### ISSUE THREAD

**Issue by patricktan98** [GeekyAnts/NativeBase#3433]
## I have gone through these following points

- [X] Check latest documentation: https://docs.nativebase.io/
- [X] Check for existing open/closed issues for a possible duplicate before creating a new issue: https://github.com/GeekyAnts/NativeBase/issues
- [X] Use the latest NativeBase release: https://github.com/GeekyAnts/NativeBase/releases
- [X] Check examples from NativeBase KitchenSink https://github.com/GeekyAnts/NativeBase-KitchenSink
- [X] For discussion purpose make use of NativeBase Slack: http://slack.nativebase.io/
- [X] For queries related to theme, check [Theme Variables](https://docs.nativebase.io/docs/ThemeVariables.html) from Docs and live NativeBase Theme Editor http://nativebase.io/customizer/

#### Expected behaviour
The Item picker should view without warnings.
#### Actual behaviour
When Item Picker is shown, the 'VirtualizedLists should never be nested inside plain ScrollViews with the same orientation - use another VirtualizedList-backed container instead.' warning shown.

#### Steps to reproduce
```
<Item picker>
                        <Picker
                            mode="dropdown"
                            iosIcon={<Icon name="arrow-down" />}
                            style={{ width: undefined }}
                            placeholder="All"
                            Header="Transaction Type"
                            iosHeader="Transaction Type"
                            placeholderStyle={{ color: "#bfc6ea" }}
                            placeholderIconColor="#007aff"
                            selectedValue={this.state.selected2}
                            onValueChange={this.onValueChange2.bind(this)}
                        >
                            <Picker.Item label="Payment" value="Payment" />
                            <Picker.Item label="Top up" value="Topup" />
                            <Picker.Item label="Refunded" value="Refunded" />
                        </Picker>
                    </Item>
```
Video Link: https://user-images.githubusercontent.com/50147719/106872277-14bbff00-670e-11eb-9497-4407ca5ac2cb.mov

#### Is the bug present in both iOS and Android or in any one of them?
The bugs occur in iOS and Android. I have tested in iOS 14.4 and Android 11.0
#### Any other additional info which would help us debug the issue quicker.

###Important

If you want your issue to be looked at quicker, attach a snack reproducible with your issue. Makes it easier for us!

**Comment by dlenskyi** [comment]
Same here
I will provide you more details from Element inspector
So basically thats because Picker component renders FlatList inside VirtualizedList (screenshot below)

<img width="384" alt="Снимок экрана 2021-02-09 в 18 29 30" src="https://user-images.githubusercontent.com/44784628/107395405-71039200-6b05-11eb-90b1-34e7860c9152.png">
<img width="411" alt="Снимок экрана 2021-02-09 в 18 29 49" src="https://user-images.githubusercontent.com/44784628/107395409-7365ec00-6b05-11eb-9d09-c870d9185833.png">

**Comment by dlenskyi** [comment]
Same issue here #3334

**Comment by ELiHimself** [comment]
Same issue for me, still waiting for solutions?

**Comment by gregormelhorn** [comment]
Please fix this and accept the pull from dlferro!

**Comment by megacherry** [comment]
yes, please accept the pull to resolve this issue.

**Comment by hoekma** [comment]
@Ajamuar  I am also still having this problem in NativeBase v2 with the sample code.  I realize you all are focused on v3 right now, but I hope you will continue to support v2 for a while until we get a chance to rewrite our app. 

```
 <Container>
      <Header />
      <Content>
        <Form>
          <Item>
            <Input value="test" onChangeText={() => true} />
          </Item>
          <Item picker>
            <Picker
              mode="dropdown"
              iosIcon={<Icon name="arrow-down" />}
              style={{ width: undefined }}
              placeholder="Select your SIM"
              placeholderStyle={{ color: '#bfc6ea' }}
              placeholderIconColor="#007aff"
              selectedValue={selected2}
              onValueChange={() => true}
            >
              <Picker.Item label="Wallet" value="key0" />
              <Picker.Item label="ATM Card" value="key1" />
              <Picker.Item label="Debit Card" value="key2" />
              <Picker.Item label="Credit Card" value="key3" />
              <Picker.Item label="Net Banking" value="key4" />
            </Picker>
          </Item>
        </Form>
      </Content>
    </Container>
```

**Comment by moseslucas** [comment]
I hope the team at native base continue support for nativebase v2 and fix this issue. There are still many apps out there that uses v2

**Comment by kleydon** [comment]
It isn't stale, its still an issue.

**Comment by cristianmacedo** [comment]
It is still an issue for me either, I have a `Select` component inside a `ScrollView` and it shows the error, but only on the iOS simulator. I tried passing `scrollEnabled: false;` and `disableVirtualization: true` to the `_actionSheetBody` property, but It makes no difference, the error keeps showing, however, everything works normally, it just slows down the screen loading because of the multiple errors (I have multiple Select components on the same screen) thrown.

**Comment by jmgarciasanti** [comment]
Yes, still an issue. It's thrown as error on iOS devices after migrating app to Expo SDK 48. The error is shown in every screen I have with a Form including a "Select" Native Base component. The error is not thrown on Android.
